lib/components/link.vue

```diff
@@ -16,7 +16,7 @@
     export default {
         computed: {
             componentType() {
-                return (this.$router && this.to) ? 'router-link' : 'a';
+                return (this.$router && this.to && !this.disabled) ? 'router-link' : 'a';
```

Comment:
Active class is added to router-link's attributes which highlights current item. so i think removing is just a good idea